var/spack/repos/builtin/packages/graphblast/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Graphblast(MakefilePackage, CudaPackage):
+    """High-Performance Linear Algebra-based Graph Primitives on GPUs"""
+
+    homepage    = "https://github.com/gunrock/graphblast"
+    git         = "https://github.com/gunrock/graphblast.git"
+
+    version('master', submodules=True)
+
+    variant('cuda', default=True)
+
+    depends_on('boost +program_options')
+
+    # This package is confirmed to compile with gcc <= 5.4.0,boost@1.58,cuda@9
```

Comment:
Is it confirmed _not_ to compile with GCC 5.5+?

Reply:
Yep! It doesn't look like the package is maintained much. I would guess that compiling with later versions of gcc is just a matter of adding or removing library include statements in the code.